sensitivity-api/src/main/java/com/powsybl/sensitivity/SensitivityComputationResults.java

```diff
@@ -41,23 +51,34 @@
     @JsonProperty("values")
     private final List<SensitivityValue> sensitivityValues;
 
+    @JsonProperty("contingencies present")
+    private final boolean contingenciesPresent;
```

Comment:
Not sure having this field is a good idea. It could lead to inconsistent data (e.g. empty contingencies map and `contingenciesPresent` as `true` or non-empty contingencies map and `contingenciesPresent` as `false`. Why do you need this field? Isn't it enough to check that `sensitivityValueContingencies` is empty or not?